.azure-pipelines/run-test-elastictest-template.yml

```diff
@@ -192,6 +198,14 @@ steps:
 
         rm -f new_test_plan_id.txt
 
+        ptf_py3=${{ parameters.PTF_PY3 }}
+        # with no value mixed image is used
+        PTF_IMAGETAG=""
+        if [ $ptf_py3 == "True" ]
```

Comment:
It's usually better to double quote the variable in string test expression, for example:
```bash
  if [ "ptf_py3" == "True" ]
```